### ISSUE TITLE
Show WebGL errors for loaded modules

### ISSUE THREAD

**Issue by TimPietrusky** [vcync/modv-3#134]
In modV2 when you load a module that has any kind of error in the code, you will only see this message in the dev console:

```
WebGL: INVALID_OPERATION: useProgram: program not valid.
```

This doesn't help at all while debugging and forces the user to use another editor to debug their code. 

## Desired Behavior

When there is an error, like a syntax error, it should show in the console what exactly is broken.

## Dependency

Relates to #83 as when you load 3rd-Party modules and you had to change them to work in modV, it's almost impossible to find out what is wrong.

**Comment by 2xAA** [comment]
ISF handles this well.

We should take a look in the ISF-renderer package and see how they handle this.

**Comment by TimPietrusky** [comment]
Yeah that is true, it would also be nice if we would see which module has the problem, as I see a lot of ISF-renderer errors when modV starts, but I have no idea which module they belong too. I added this in the the main description!

**Comment by 2xAA** [comment]
You can already see which has the problem.
modV shows which modules can't be registered in the console.

vcync/modv-3#170 even mentions this.

**Comment by TimPietrusky** [comment]
What I see is some WebGL errors, then the source-code of some modules (looks like GLSL) and then the two error messages like in vcync/modv-3#170. I have no idea which module the errors belong to if there are multiple problems.
